Automattic/wp-calypso
# Masterbar: replace 'Edit Profile' and 'My Account' with 'My Profile'

Related to:

- https://github.com/Automattic/wp-calypso/issues/95055
- https://github.com/Automattic/wp-calypso/issues/94470

Jetpack counterpart:

- https://github.com/Automattic/jetpack/pull/39600

## Proposed Changes

This PR modifies the following submenus in the `Howdy, {display menu}` menu in the admin bar:

- Point the `Edit Profile` submenu to `/me` always (and rename it to `My Profile` to match the page name).
- Remove the `My Account` submenu.

|Before|After|
|-|-|
|<img width="261" alt="image" src="https://github.com/user-attachments/assets/5586e924-05ae-4c10-9dee-e53a353ebd79">|<img width="261" alt="image" src="https://github.com/user-attachments/assets/df5d60ce-fcf6-48f1-9e0c-7b958221f8ca">|

~~In return, this PR proposes to add a link back to wp-admin's `profile.php` for the current site as follows:~~

EDIT: We decided not to include the following in this PR.

<img width="909" alt="image" src="https://github.com/user-attachments/assets/ff7b8532-7c88-4646-9d32-b2af6b49311a">


## Why are these changes being made?

The `profile.php` wp-admin page that the `Edit Profile` currently points to, contains mostly links back to Calypso. The most useful fields (display names, language, email) can only be done at user level in Calypso. So, to reduce jumps between Calypso <> wp-admin interface, this PR proposes that the submenu should point to Calypso.


## Testing Instructions

1. Go to `/home/:site`
1. Click the Howdy menu; verify it points to `/me`.
1. Click the Howdy -> My Profile submenu; verify it points to `/me`.
2. In `/me`, verify you can see the notice and it goes to the site's `profile.php` when clicked.

## Pre-merge Checklist
